.gitignore

```diff
@@ -4,3 +4,4 @@
 *.pyc
 config.py
 *.log
+.*.swp
```

Comment:
I would suggest to use a [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) file for things that are not directly related to the project.